src/filters/path.js

```diff
@@ -7,10 +7,13 @@ module.exports = function(fractal) {
 
     return function(path) {
 
-        let env = this.context._env;
-        let request = env.request || this.context._request;
+        if(!this.context._env || this.context._env.server) {
+            return path;
+        } else {
```

Comment:
Just a stylistic thing, but we can drop the `else` here. We're already returning if the problem properties don't exist.